src/sdam/server.ts

```diff
@@ -151,31 +153,36 @@ export class Server extends TypedEventEmitter<ServerEvents> {
       this.clusterTime = clusterTime;
     });
 
-    // create the monitor
-    this[kMonitor] = new Monitor(this, this.s.options);
+    if (!this.loadBalanced) {
```

Comment:
Could this be rewritten as `if (this.loadBalanced) return`? It looks like all the change here is indentation but just want to make sure

Reply:
I've changed per suggestion.